src/main/java/com/amazon/opendistroforelasticsearch/knn/index/util/NmsLibVersion.java

```diff
@@ -20,14 +20,14 @@
     /**
      * Latest available nmslib version
      */
-    V206("206"){
+    V208("208"){
         @Override
         public String indexLibraryVersion() {
-            return "KNNIndexV2_0_6";
+            return "KNNIndexV2_0_8";
         }
     };
 
-    public static final NmsLibVersion LATEST = V206;
+    public static final NmsLibVersion LATEST = V208;
```

Comment:
why can't we create a "latest" as enum value, so that we don't have to update this line every time. What do you think?

Reply:
I think we could but Im not sure how it would be better. The purpose of using the versioning is to handle potential backwards compatibility issues if we upgrade to an nmslib version that breaks backwards compatibility. We havent had this issue so far, so we just replace the nmslib version with the new one. 

However, if we end up having this issue, it seems to me, LATEST should just be a pointer to an enum value and enum values should be concrete versions.